src/Connections/GitlabRepo.ts

```diff
@@ -222,12 +222,16 @@ export class GitLabRepoConnection extends CommandConnection {
             return;
         }
         const branchname = event.ref.replace("refs/heads/", "");
-        const commitsurl = `${event.project.homepage}/-/commits/${branchname}`
+        const commitsurl = `${event.project.homepage}/-/commits/${branchname}`;
         const branchurl = `${event.project.homepage}/-/tree/${branchname}`;
         const shouldName = !event.commits.every(c => c.author.name === event.commits[0]?.author.name);
 
+        const maxCommits = 5;
```

Comment:
if this is a const, just define it at the top of the file as `PUSH_MAX_COMMITS` or something.

Reply:
it's just a helper for now, it should eventually be read from room state #329

Reply:
Yeah we'd likely still want a default.